### DISCUSSION TITLE
Import Artisan CSV... Coordinate shift

### DISCUSSION THREAD

**Discussion by Terracotta-6** [artisan-roaster-scope/artisan#1357] [General]
Export Artisan CSV... after Import Artisan CSV... 
![74a12ea0ce04f4bd4b0d1d97df3568d](https://github.com/artisan-roaster-scope/artisan/assets/149906183/f7b2a772-6ceb-4e7b-a074-8b562aa5537a)
[test.csv](https://github.com/artisan-roaster-scope/artisan/files/13732894/test.csv)

**Comment by Terracotta-6** [comment]
Continuous build 46ed365

**Comment by MAKOMO** [comment]
Thanks for your report. It turned out that this is caused by an issue with the CSV export which is fixed now (Commit 46ed365d776e71fbde66e48167c0be938d4dfcb5). New builds including this fix should be online within an hour.
